src/main/io/serial_cli.c

```diff
@@ -500,6 +500,10 @@ static const char * const lookupTableLowpassType[] = {
     "NORMAL", "HIGH"
 };
 
+static const char * const lookupTableLedAux[] = {
+    "OFF", "1","2","3","4"
```

Comment:
In previous versions of betaflight, many values were replaced by more meaningful values using lookup tables. I think the maintainers can decide whats the best way.